File: Pemrograman-1/Modul 5/C/PRAK501-2310817210014-DamarjatiSuryoLaksono.c
#include <stdio.h>
//Buatlah Function Disini

int MaxBilangan(int a,int b,int c, int d){
    int hasbi = a;
    
    if (b>hasbi){
        hasbi = b;
    }
    if (c>hasbi){
        hasbi = c;
    }
    if (d>hasbi){
        hasbi = d;
    }
    return hasbi ;
}
    int main() {
    int a, b, c, d;
    scanf("%d %d %d %d", &a, &b, &c, &d);
    int hasil = MaxBilangan(a, b, c, d);
    printf("%d", hasil);
return 0;
}

File: Pemrograman-1/Modul 5/PY/PRAK501-2310817210014-DamarjatiSuryoLaksono.py
#Buatlah Function Disini
def Maxbilangan (a,b,c,d) :
    max = a
    if max<b:
        max = b
    if max<c:
        max = c
    if max<d:
        max = d
    return max 

a, b, c, d = map(int,input("").split(" "))
hasil = Maxbilangan(a,b,c,d)
print(hasil)